### PR TITLE
odb/gui: coverity fixes

### PR DESCRIPTION
**Pull request by gadfort**
Added:
- `const std::string& arg` to code generator in ODB since that seemed to be a common warning from the SWIG generated code.
- defaults to the two classes identified.

### CODE REVIEW

**Review by maliberty — APPROVED** [review]

File: src/gui/src/scriptWidget.cpp [hunk 1]
```diff
@@ -139,11 +139,11 @@ ScriptWidget::ScriptWidget(QWidget* parent)
 // the logging, so, for reports, we use a buffer.
 void ScriptWidget::flushReportBufferToOutput()
 {
+  std::lock_guard guard(reporting_);
```
Comment (maliberty) [inline src/gui/src/scriptWidget.cpp:142]: I'm assuming you meant to include this

Comment (gadfort) [reply src/gui/src/scriptWidget.cpp:142]: Yeah, it was in the list and small enough to be quick

**Comment by gadfort** [comment]
@maliberty I'm not sure what makes my PRs special to this segfault seems like there is an issue in sta or something (but unrelated to the PR):
```
repair_timing -skip_gate_cloning
Signal 11 received
Stack trace:
 0# 0x00005E405777A30B in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 1# 0x00007CA4D828E520 in /lib/x86_64-linux-gnu/libc.so.6
 2# sta::ClkInfo::refsFilter(sta::StaState const*) const in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 3# sta::TagGroupBldr::setMatchArrival(sta::Tag*, sta::Tag*, float const&, int, sta::PathVertexRep*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 4# sta::ArrivalVisitor::visitFromToPath(sta::Pin const*, sta::Vertex*, sta::RiseFall const*, sta::Tag*, sta::PathVertex*, sta::Edge*, sta::TimingArc*, float, sta::Vertex*, sta::RiseFall const*, sta::Tag*, float&, sta::MinMax const*, sta::PathAnalysisPt const*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 5# sta::PathVisitor::visitFromPath(sta::Pin const*, sta::Vertex*, sta::RiseFall const*, sta::PathVertex*, sta::Edge*, sta::TimingArc*, sta::Pin const*, sta::Vertex*, sta::RiseFall const*, sta::MinMax const*, sta::PathAnalysisPt const*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 6# sta::PathVisitor::visitEdge(sta::Pin const*, sta::Vertex*, sta::Edge*, sta::Pin const*, sta::Vertex*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 7# sta::PathVisitor::visitFaninPaths(sta::Vertex*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 8# sta::ArrivalVisitor::visit(sta::Vertex*) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
 9# 0x00005E4057AB884D in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
10# sta::DispatchQueue::dispatch_thread_handler(unsigned long) in /tmp/workspace/OpenROAD-Public_PR-5524-merge/build/src/openroad
11# 0x00007CA4D865A253 in /lib/x86_64-linux-gnu/libstdc++.so.6
12# 0x00007CA4D82E0AC3 in /lib/x86_64-linux-gnu/libc.so.6
13# 0x00007CA4D8372850 in /lib/x86_64-linux-gnu/libc.so.6
ERROR segmentation violation
```

**Comment by maliberty** [comment]
We turned off sta threads due to similar issues.  Are you reasonably up to date with master?

**Comment by gadfort** [comment]
@maliberty I'm 14 commits behind the master (including these 4)

**Comment by maliberty** [comment]
That should be new enough.

**Pull request merged**